docs/apriori_sbms.ipynb

```diff
@@ -0,0 +1,384 @@
+    "fig.set_size_inches(16, 32)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once again, we find that cell type explains the topology the best. This is still not entirely surprising, since the\n",
+    "connectivity of interneurons should be very different than that of sensory or motor neurons"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "## C. Elegans Male and Hermaphrodite Conectomes (Worm Wiring) ##"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# load chemical synapse connectome files\n",
+    "path_h = '../json_connectomes/worm_wiring/connectome/Hermaphrodite/0.json'\n",
+    "path_m = '../json_connectomes/worm_wiring/connectome/Male/0.json'\n",
+    "ww_connectomes = []\n",
+    "ww_connectomes.append(GraphIO.load(path_h)[0])\n",
+    "ww_connectomes.append(GraphIO.load(path_m)[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "groups = [[\"hemisphere\"], [\"cell_type0\"], [\"hemisphere\", \"cell_type0\"]]\n",
+    "types = ['Hermaphrodite', 'Male']\n",
+    "rows = {}\n",
+    "for i in range(len(ww_connectomes)):\n",
+    "    g_row = {}\n",
+    "    for group in groups:\n",
+    "        g_row[str(group)] = fit_apriori_sbm(ww_connectomes[i], group, plot_sbms=False)\n",
+    "    rows['C. Elegans ' + types[i]] = g_row\n",
+    "ww_results = pd.DataFrame.from_dict({(i,j): rows[i][j]\n",
+    "                           for i in rows.keys()\n",
+    "                           for j in rows[i].keys()},\n",
+    "                       orient='index')\n",
+    "ww_results.head(6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One last time, we'll plot BIC and Likelihood"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots(2, 2)\n",
+    "out_index = ww_results.index.levels[0]\n",
+    "colors = ['red', 'blue']\n",
+    "for i, key in enumerate(out_index):\n",
+    "    data = ww_results.loc[key]\n",
+    "    ax[i, 0].scatter(data['bic'].index, data['bic'], label=key, color=colors[i])\n",
+    "    ax[i, 1].scatter(data['liklihood'].index, data['liklihood'], label=key, color=colors[i])\n",
+    "    ax[i, 0].set(xlabel='Groups, ' + types[i], ylabel='-BIC')\n",
+    "    ax[i, 1].set(xlabel='Groups, ' + types[i], ylabel='Likelihood')\n",
+    "plt.ylabel('-BIC \\t Likelihood')\n",
+    "fig.suptitle('BIC and Likelihood: Worm Wiring C. Elegans Hermaphrodite and Male')\n",
```

Comment:
I think these graphs should be combined with the hermaphrodite/male having different shapes (circle vs triangle).  Also, they should be green to match the format spec.

Reply:
Same issue with scaling as above, but will change the colors